agent-connection/src/main/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/AgentWebSocketClient.kt

```diff
@@ -5,9 +5,10 @@ import com.luxoft.blockchainlab.hyperledger.indy.utils.SerializationUtils
 import mu.KotlinLogging
```

Comment:
What relations this PR has with Sergey's PR? It should be merged before or after?

Reply:
Because of this change it should be merged after